src/main/java/frc/robot/commands/grabber/RollerIntakeCommand.java

```diff
@@ -24,7 +24,7 @@ public void initialize() {
 
     @Override
     public void execute() {
-        rollerSubsystem.setRollPower(0.3);
+        rollerSubsystem.setRollPower(-0.3);
```

Comment:
i think we should have negative power spit out and positive power suck in... is that not currently the case?